app/assets/stylesheets/file.css

```diff
@@ -161,6 +170,21 @@
   }
 }
 
+/* The background applied relies on the number of rows displayed in the file viewer */
+.stripes-background {
+  height:100%;
```

Comment:
@hudajkhan I think this needs to be calculated in the js. Right now it is making the viewer scroll when there is only two items